tests/unit_tests/restruct_poc/system/test_integrate_stiffness_matrix.cpp

```diff
@@ -496,8 +452,8 @@ void TestIntegrateStiffnessMatrix_1Element1Node2QPs(
     const Kokkos::View<const double[2][6][6]>& qp_Ouu,
     const Kokkos::View<const double[2][6][6]>& qp_Quu, const std::array<double, 36>& exact_M_data
 ) {
-    constexpr auto number_of_nodes = 1;
-    constexpr auto number_of_qps = 2;
+    constexpr auto number_of_nodes = size_t{1U};
+    constexpr auto number_of_qps = size_t{2U};
```

Comment:
Would this be cleaner?

```
constexpr size_t number_of_nodes{1};
constexpr size_t number_of_qps{2};
```

Reply:
Depends on who you ask.  I like using auto and making type defined by what appears on the right-hand side of the expression.  This helps keywords and variable names line up better for complex types.  It also makes the form of the code uniform regardless of your initialization strategy (copy assignment vs constructor vs factor function all look the same: `auto var_name = make_my_var()`).  I admit it looks a little chunky for really simple uses like this, but I like that it provides one uniform and organized way to declare a variable in all cases and all variable types.  

I think C++20 or 23 gives us a literal for `size_t` like the U for `unsigned`.  Once we can use that, it will make this type of code look better.